missal1962/ical.py

```diff
@@ -22,6 +22,39 @@ def build(days: Dict[datetime.date, Day], rank: int, lang: str) -> str:
             event.add("dtstart", datetime_)
             event.add("dtstamp", now)
             event.add("uid", f"{datetime_.strftime('%Y%m%d')}@missalemeum.com")
-            event.add("description", "https://www.missalemeum.com/{}/{}".format(lang, datetime_.strftime("%Y-%m-%d")))
+
+            # Map color arrau to an array of human readable strings
+            color_map = {
+              "w": "white",
+              "r": "red",
+              "b": "black",
+              "v" : "violet",
+              "g" : "green",
+            }
+
+            colors = []
+            for c in celebration.colors:
+                colors.append(color_map[c])
+
+            # Map ranks to human readable strings
+            rank_map = {
+              1: "First Class Feast",
+              2: "Second Class Feast",
+              3: "Third Class Feast",
+              4 : "Feria",
```

Comment:
Internationalization and refactoring needed in a same way as in case of `color_map`.